visualization/dashboard/src/Localization/en.json

```diff
@@ -482,5 +485,23 @@
         "inTheRangeOf": "[ {0} ]",
         "overflowFilterArgs": "{0} and {1} others",
         "_overflowFilterArgs.comment": "first placeholder is the first one or two items in a long list, the second placeholder is the count of remaining items"
+    },
+    "Statistics": {
+        "mse": "MSE: {0}",
+        "_mse.comment": "the mean squared error, see https://en.wikipedia.org/wiki/Mean_squared_error",
+        "rSquared": "R-squared: {0}",
+        "_rSquared.comment": "the coefficient of determination, see https://en.wikipedia.org/wiki/Coefficient_of_determination",
+        "meanPrediction": "Mean prediction {0}",
+        "_meanPrediction.comment": "the average of all the predictions",
+        "accuracy":"Accuracy: {0}",
+        "_accuracy.comment": "computed accuracy of model on a subgroup, see https://en.wikipedia.org/wiki/Evaluation_of_binary_classifiers",
```

Comment:
do we have multiclass metrics too?  like micro and macro averaged precision/recall for example?

Reply:
Mehrnoosh is compiling a set of default metrics to show in the multiclass case. This PR is to add support for regression case, we already support binary. Multiclass is still being designed.